src/main/java/com/dinosaur/dinosaurexploder/view/DinosaurMenu.java

```diff
@@ -5,35 +5,117 @@
+                    JSONObject data = (JSONObject) parser.parse(new FileReader("src/main/resources/assets/translation/german.json"));
+                    startButton.setText(data.get("start").toString());
+                    quitButton.setText(data.get("quit").toString());
+
+                } catch (Exception e) {
+                    e.printStackTrace();
+                }
+            } else if (t1.equals("japanese")) {
+                JSONParser parser = new JSONParser();
+                JSONObject japan= null;
+                try {
+                    japan = (JSONObject) parser.parse(new FileReader("src/main/resources/assets/translation/japanese.json"));
+                } catch (IOException e) {
+                    throw new RuntimeException(e);
+                } catch (ParseException e) {
+                    throw new RuntimeException(e);
+                }
+                startButton.setText(japan.get("start").toString());
+                quitButton.setText(japan.get("quit").toString());
+            }
+
+            else if (t1.equals("french")) {
+                JSONParser parser = new JSONParser();
+                JSONObject japan= null;
+                try {
+                    japan = (JSONObject) parser.parse(new FileReader("src/main/resources/assets/translation/french.json"));
+                } catch (IOException e) {
+                    throw new RuntimeException(e);
+                } catch (ParseException e) {
+                    throw new RuntimeException(e);
+                }
+                startButton.setText(japan.get("start").toString());
+                quitButton.setText(japan.get("quit").toString());
+            }
+
+            else if (t1.equals("russian")) {
+                JSONParser parser = new JSONParser();
+                JSONObject japan= null;
+                try {
+                    japan = (JSONObject) parser.parse(new FileReader("src/main/resources/assets/translation/russian.json"));
+                } catch (IOException e) {
+                    throw new RuntimeException(e);
+                } catch (ParseException e) {
+                    throw new RuntimeException(e);
+                }
+                startButton.setText(japan.get("start").toString());
+                quitButton.setText(japan.get("quit").toString());
+            }
+            else if (t1.equals("english")) {
+                JSONParser parser = new JSONParser();
+                JSONObject japan= null;
+                try {
+                    japan = (JSONObject) parser.parse(new FileReader("src/main/resources/assets/translation/en.json"));
+                } catch (IOException e) {
+                    throw new RuntimeException(e);
+                } catch (ParseException e) {
+                    throw new RuntimeException(e);
+                }
+                startButton.setText(japan.get("start").toString());
+                quitButton.setText(japan.get("quit").toString());
+            }

```

Comment:
Could you refactor this code please ? It is very redundant and if I want to add a new language it will be a mess.
Maybe create a new file to code clean and have just ONE responsibility.